plugins/flatpak/gs-flatpak.c

```diff
@@ -4062,9 +4057,16 @@ gs_flatpak_update_app (GsFlatpak *self,
 		}
 	}
 
-	if (!gs_flatpak_has_space_to_update (self, app, list)) {
-		g_debug ("Skipping automatic update for %s: not enough space on disk",
-			 gs_app_get_unique_id (app));
+	is_auto_update = gs_utils_app_is_auto_updating (app);
+	if (!gs_flatpak_has_space_to_update (self, app, list, is_auto_update)) {
+		if (is_auto_update)
+			g_debug ("Skipping automatic update for %s: not enough space on disk",
+				 gs_app_get_unique_id (app));
+		else
+			g_debug ("Skipping update for %s: not enough space on disk",
+				 gs_app_get_unique_id (app));
+
+		gs_app_set_state_recover (app);
```

Comment:
You shouldn't return `FALSE` without setting the `error` output parameter.

Reply:
Then it would show up in the error banner, no? I think we want to avoid that case basically.

Reply:
Ah, because this is the skip (sorry I was in a rush). In this case then you have to return `TRUE`, as this is the value plugin methods return also when they ignore a request.